### PR TITLE
Add auto author assign at PR #35

### LINKED ISSUE #35: PR author gets auto-assigned
#### Description

In most cases, pull request author should be assigned an assignee of the pull request. This task makes sure the PR author assigns as an assignee.

#### Acceptance Criteria

- [ ] Install https://github.com/marketplace/actions/auto-author-assign
- [ ] Test it

Complexity: 2

### PR DESCRIPTION
**Pull request by JohnGavr**
Add the workflow yaml file for auto assign the author of the PR as reference to #35.

Tested in personal repo and works fine.

### CODE REVIEW

File: .github/workflow/auto-author-assign.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: 'Auto Author Assign'
+on:
+  pull_request:
+    types: [opened, reopened]
+jobs:
+  add-assignees:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Test toshimaru/auto-author-assign action
```
Comment (drpaneas) [inline .github/workflow/auto-author-assign.yml]: Can we change that to a different `- name:` or it has to be this specific one?

Comment (cerebrux) [reply .github/workflow/auto-author-assign.yml]: Yes, you can put whatever you want in the `-name` tag

Comment (drpaneas) [reply .github/workflow/auto-author-assign.yml]: Then let's change it to something less bloated.

```suggestion
      - name: auto-author-assign
```

**Review by drpaneas — CHANGES_REQUESTED** [review]

**Review by cerebrux — APPROVED** [review]

**Review by drpaneas — APPROVED** [review]
LGTM

**Pull request merged**